src/components/tabs/demoStretchHeight/index.html

```diff
@@ -0,0 +1,35 @@
+<div ng-controller="DemoStretchHeightCtrl" ng-cloak>
+  <md-block style="height: 460px" layout="column">
+    <p class="md-padding">The content in this <code>&lt;p&gt;</code> tag and the <code>md-tab</code> element below are in a 460 pixel tall container. <code>md-stretch-height</code> will automatically account for sibling elements.</p>
```

Comment:
Let's move the `md-padding` up one level to be consistent with the other demos. This worked for me locally:
```html
  <div class="md-padding" style="height: 460px" layout="column">
    <p>The content in this <code>&lt;p&gt;</code> tag and the <code>md-tab</code> element below are in a 460 pixel tall container. <code>md-stretch-height</code> will automatically account for sibling elements.</p>
    <md-tabs md-stretch-height>
```

Reply:
@Splaktar Doing so caused a rendering bug in the demo for me locally so this is a change outside what I'm comfortable adding.